server/container/guice/guice-common/src/main/java/org/apache/james/modules/server/DropWizardMetricsModule.java

```diff
@@ -91,8 +82,7 @@ public DropWizardInitializer(DropWizardMetricFactory dropWizardMetricFactory, Dr
             this.dropWizardJVMMetrics = dropWizardJVMMetrics;
         }
 
-        @Override
-        public void configure(HierarchicalConfiguration config) throws ConfigurationException {
+        public void start() {
```

Comment:
while we are at it, can someone explain me why Startable interface does not define start method?

Reply:
Because some classes requires arguments to actually be starting?

Reply:
I don't get it. I did not find any start() method in a Startable implementation with arguments. But I did find some  Startable implementations without start() method.

Reply:
Well good question. @mbaechler might be able to answer this better than me.

Reply:
Startable is a marker interface that allows to enforce the right types in ConfigurationPerformer :+1: 
```java
List<Class<? extends Startable>> forClasses()
```
and in StartablesModule:
```java
    private <I> void hear(TypeLiteral<I> type, TypeEncounter<I> encounter) {
        InjectionListener<? super I> injectionListener = ignored -> startables.add(type.getRawType().asSubclass(Startable.class));
        encounter.register(injectionListener);
    }
```